src/test/java/seedu/address/logic/parser/AddCommandParserTest.java

```diff
@@ -47,7 +48,7 @@
 
 public class AddCommandParserTest {
     private AddCommandParser parser = new AddCommandParser();
-
+    @Disabled
```

Comment:
Relevant use of @disabled annotation.

Reply:
We need to raise an issue to reenable these so that we don't forget later on